### PR TITLE
README.md but changed instructions for jekyll

### PR DESCRIPTION
**Pull request by kennedy**
The readme file was missing in my original PR. I am bringing it back, including various other fixes for a jekyll specific site

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,38 +1,37 @@
-Contributing to FOSSProfiles
-============================
+# Contributing to FOSSProfiles
```
Comment (kennedy) [inline .github/CONTRIBUTING.md]: My markdown linter does not like different styles of heading in the same file. I think this helps with consistency anyway

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
 as a base for making your own profile.
 
-#### Required fields
+### Required fields
 
 All profiles are required to have these fields completed:
 
 * `name`
-* `website`
+* `website` (please include `http[s]://`)
```
Comment (kennedy) [inline .github/CONTRIBUTING.md]: What is the best way to illustrate this requirement?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -44,7 +43,6 @@ everything still works.
 ./test.sh
```
Comment (kennedy) [inline .github/CONTRIBUTING.md]: Im not sure how to handle this portion. I updated the `test.sh` script to the appropriate directory now

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (jwflory) [reply .github/CONTRIBUTING.md]: @kennedy I think this works for now.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (jwflory) [reply .github/CONTRIBUTING.md]: @kennedy We might want to adopt a build script like the one for the FOSS website: https://github.com/FOSSRIT/fossrit.github.io/blob/master/build.sh

**Pull request merged**